config/initializers/loa.rb

```diff
@@ -8,5 +8,5 @@ module LOA
 
   IDME_LOA1 = 'http://idmanagement.gov/ns/assurance/loa/1/vets'
   IDME_LOA2 = 'http://idmanagement.gov/ns/assurance/loa/2/vets'
-  IDME_LOA3 = 'http://idmanagement.gov/ns/assurance/loa/3/vets'
```

Comment:
The change to `IDME_LOA3` works for both v0 and v1.  The IAM team requested that we make this change for SSOe authentication, however I'm not sure why they didn't make the same ask for `IDME_LOA1`.  I'll work on getting a better explanation from their team and post that on this PR.

Reply:
I've ran tests locally using the `v0` controller and the new IDME_LOA3 works as expected.  To add some context, the change was made because SSOe already recognised the AuthNContext urls without the `/vets` suffix, so ID.ME made a change to support both.  Going forward we could probably use either of the 2 urls with the `v0` controller, but I'm guessing they might deprecate `http://idmanagement.gov/ns/assurance/loa/3/vets` sometime in the future.